src/include/sof/samples/audio/c_models_inc/process.h

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Mihai Despotovici <mihai.despotovici@nxp.com>
+ */
+
+#ifndef __PROCESS_H__
+#define __PROCESS_H__
+
+#include "config.h"
+
+void process_data(
+	uint8_t preprocessed_data[CONFIG_PREPROCESSED_SIZE],
+	uint8_t confidences[4]
```

Comment:
4 is a magic number. Add a define for it and explain why is 4 the chosen number. :)

Reply:
@cristinailie also we can go with `process_data` prototype definition on the same line. Limit is at 100 chars / line.